sdk/servicebus/service-bus/src/clientEntityContext.ts

```diff
@@ -200,26 +219,78 @@ export namespace ClientEntityContext {
       }
     };
 
-    (entityContext as ClientEntityContext).clearClientReference = async (clientId: string) => {
-      delete context.clients[clientId];
+    const isManagementClientSharedWithOtherClients = (): boolean => {
+      let isManagementClientShared = false;
```

Comment:
nit: you could get rid of the variable and simply `return true`/`return false` here.

Reply:
fair point, updated